game-core/src/main/java/games/strategy/engine/data/ChangeAttachmentChange.java

```diff
@@ -59,13 +58,16 @@ public ChangeAttachmentChange(
       final Object newValue,
       final Object oldValue,
       final String property,
-      final boolean resetFirst) {
-    this.attachmentName = attachmentName;
+      final boolean clearFirst) {
+    this.attachmentName =
+        Optional.ofNullable(attachmentName)
+            .map(name -> name.replaceAll("ttatch", "ttach"))
```

Comment:
I mean we all know why this exists, but it would be nice documenting it here in the code nevertheless

Reply:
Same on other places

Reply:
Added in: https://github.com/triplea-game/triplea/pull/7661